src/colvarscript.cpp

```diff
@@ -480,13 +481,36 @@ int colvarscript::proc_bias(colvarbias *b, int objc, unsigned char *const objv[]
     if (subcmd == "count") {
       int index;
       if (!(std::istringstream(param) >> index)) {
-        result = "bin_count: error parsing bin index";
+        result = "bias count: error parsing bin index";
         return COLVARSCRIPT_ERROR;
       }
       result = cvm::to_str(b->bin_count(index));
       return COLVARS_OK;
     }
 
+    // Get ABF gradient
+    if (subcmd == "gradient") {
```

Comment:
This command should have a more specific name, e.g. `bin_gradient`, and be the gradient of the bias energy, so perhaps `bin_forces` would be even more appropriate.

More generally, it seems reasonable to me to have `energy` take as optional arguments  the values of collective variables (other than the current).  `bin_energy` could then be the corresponding function that takes bin indices instead of real values.  Likewise, `forces` and `bin_forces` would do the same in gradient space.

You should not provide a complete implementation of the above, but at least use command names that do not clash with potential use in restraints or metadynamics biases.

Reply:
Sure, I can call it "bin_forces". For consistency, maybe I should make "bin_count" an alias for "count", while maintaining "count" for backwards compatibility?

I shouldn't have a problem implementing "bin_energy", "bin_forces" and "forces" once I get the other issues worked out.

Reply:
That sounds reasonable: `count` itself would be difficult to define for real-valued argument (it needs an interval to compute a proper count).  But `bin_count` would definitely go along well.